packages/admin-ui/src/pages/dashboard/components/HostStats.tsx

```diff
@@ -52,17 +53,19 @@ export function HostStats() {
   const cpuStats = useApi.statsCpuGet();
   const memoryStats = useApi.statsMemoryGet();
   const diskStats = useApi.statsDiskGet();
+  const hostUptime = useApi.getHostUptime();
 
   useEffect(() => {
     const interval = setInterval(() => {
       cpuStats.revalidate();
       diskStats.revalidate();
       memoryStats.revalidate();
+      hostUptime.revalidate();
```

Comment:
I would set more time to revalidate the uptime, not necessary to reload the data ever 5 s. Use another useffect with more time on it

Reply:
Changed to 1 minute